apps/frontend/src/app/5_pages/MarketMakingPage/components/AdjustAndDepositModal/components/NewPoolStatistics/NewPoolStatistics.tsx

```diff
@@ -1,29 +1,36 @@
-import React, { FC, useMemo } from 'react';
+import React, { FC, useEffect, useMemo, useState } from 'react';
 
+import classNames from 'classnames';
+import { ethers } from 'ethers';
 import { t } from 'i18next';
 
-import { SupportedTokens } from '@sovryn/contracts';
+import { SupportedTokens, getTokenContract } from '@sovryn/contracts';
+import { getProvider } from '@sovryn/ethers-provider';
 import { SimpleTable, SimpleTableRow } from '@sovryn/ui';
 import { Decimal } from '@sovryn/utils';
 
+import { defaultChainId } from '../../../../../../../config/chains';
+
 import { AmountRenderer } from '../../../../../../2_molecules/AmountRenderer/AmountRenderer';
+import {
+  BITCOIN,
+  BTC_RENDER_PRECISION,
+  TOKEN_RENDER_PRECISION,
+} from '../../../../../../../constants/currencies';
+import { useGetRBTCPrice } from '../../../../../../../hooks/zero/useGetRBTCPrice';
 import { translations } from '../../../../../../../locales/i18n';
-import { decimalic } from '../../../../../../../utils/math';
+import { decimalic, fromWei } from '../../../../../../../utils/math';
+import { useGetAccumulatedReward } from '../../../../hooks/useGetAccumulatedReward';
+import { useGetExpectedTokenAmount } from '../../../../hooks/useGetExpectedTokenAmount';
+import { useGetUserInfo } from '../../../../hooks/useGetUserInfo';
 import { AmmLiquidityPool } from '../../../../utils/AmmLiquidityPool';
 import { AdjustType } from '../../AdjustAndDepositModal.types';
 
-// TODO: Fetch rewards in the component. If you want to test the functionality, set it to Decimal.ONE
-const transferRewards = Decimal.ZERO; // Decimal.ONE
-
-// TODO: Fetch weekly rewards estimation in the component. If you want to test the functionality, set it to Decimal.ONE
-const weeklyRewardsEstimation = Decimal.ONE; // Decimal.ONE
-
-// TODO: Fetch initialPoolBalance in the component.
-const initialPoolBalance = decimalic(20);
-
 const pageTranslations =
   translations.marketMakingPage.adjustAndDepositModal.newPoolStatistics;
 
+const WEEKLY_REWARDS_AMOUNT = '20000';
```

Comment:
```suggestion
const WEEKLY_REWARDS_AMOUNT = Decimal.from(20000);
```
Define it as `Decimal` so you don't need to cast it later.